contracts/src/utils/TokenScalingUtils.sol

```diff
@@ -34,6 +36,19 @@ library TokenScalingUtils {
         return _scaleTokens(tokenMultiplier, multiplyOnDestination, destinationTokenAmount, false);
     }
 
+    function deriveFactors(
+        uint8 sourceTokenDecimals,
+        uint8 destinationTokenDecimals
+    ) internal pure returns (uint256 tokenMultiplier, bool multiplyOnDestination) {
+        multiplyOnDestination = destinationTokenDecimals > sourceTokenDecimals; // TODO: should this be >= ?
```

Comment:
I don't think >= or > matters here

Reply:
thanks; removed in https://github.com/ava-labs/teleporter-token-bridge/pull/142/commits/e14a1529e5331eb122cf37f9e0725ed7735d00a1